address/constants.go

```diff
@@ -1,62 +1,37 @@
 package address
 
 import (
-	"gx/ipfs/QmZo5avr9dhVVRzcpKnU9ZGQuPaU62pbufUHXBNB7GwLzQ/go-basex"
-)
-
-// HashLength is the length of an the hash part of the address in bytes.
-const HashLength = 20
-
-// Length is the lengh of a full address in bytes.
-const Length = 1 + 1 + HashLength
-
-// Version is the current version of the address format.
-const Version byte = 0
-
-// Base32Charset is the character set used for base32 encoding in addresses.
-const Base32Charset = "qpzry9x8gf2tvdw0s3jn54khce6mua7l"
+	"encoding/base32"
 
-// Base32CharsetReverse is the reverse character set. It maps ASCII byte -> Base32Charset index on [0,31].
-var Base32CharsetReverse = [128]int8{
-	-1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
-	-1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
-	-1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
-	15, -1, 10, 17, 21, 20, 26, 30, 7, 5, -1, -1, -1, -1, -1, -1,
-	-1, 29, -1, 24, 13, 25, 9, 8, 23, -1, 18, 22, 31, 27, 19, -1,
-	1, 0, 3, 16, 11, 28, 12, 14, 6, 4, 2, -1, -1, -1, -1, -1,
-	-1, 29, -1, 24, 13, 25, 9, 8, 23, -1, 18, 22, 31, 27, 19, -1,
-	1, 0, 3, 16, 11, 28, 12, 14, 6, 4, 2, -1, -1, -1, -1, -1,
-}
-
-// Base32 is a basex instance using the Base32Charset.
-var Base32 = basex.NewAlphabet(Base32Charset)
+	"gx/ipfs/QmZp3eKdYQHHAneECmeK6HhiMwTPufmjC8DuuaGKv3unvx/blake2b-simd"
+)
 
 var (
-	// TestAddress is an account with some initial funds in it
+	// TODO remove TestAddresses
```

Comment:
TODO should explain why we want to remove TestAddresses. It should also explain why we're not doing it in this PR.